common/v2/components/WalletUnlock/Web3Provider.tsx

```diff
@@ -90,6 +88,48 @@ class Web3ProviderDecrypt extends Component<Props, State> {
       this.setState({ ...this.state, web3Unlocked: false });
     }
   }
+
+  private getWeb3Footer(strWeb3Name: string) {
+    switch (strWeb3Name.toUpperCase()) {
+      default:
+      case 'METAMASK':
+        return (
+          <div className="MetaMask-footer">
+            <div>
+              {translate(`ADD_ACCOUNT_${this.state.web3ProviderSettings.name}_FOOTER`)}{' '}
+              <NewTabLink
+                content={translate(
+                  `ADD_ACCOUNT_${this.state.web3ProviderSettings.name}_FOOTER_LINK`
+                )}
+                href="https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn?hl=en"
+              />
+            </div>
+            <div>
+              <NewTabLink
+                content={translate('ADD_ACCOUNT_METAMASK_HELP')}
+                href={`${KB_URL}/how-to/migrating/moving-from-mycrypto-to-metamask`}
+              />
+            </div>
+          </div>
+        );
+        break;
+      case 'TRUST':
+        break;
+    }
+  }
+
+  private getWeb3Provider() {
+    if (window.web3) {
+      switch (true) {
+        default:
+        case window.web3.currentProvider.isMetaMask:
+          return { name: 'METAMASK', icon: MetamaskSVG };
+        case window.web3.currentProvider.isTrust:
+          return { name: 'TRUST', icon: TrustWalletSVG };
```

Comment:
Let's since the icon and the name are already defined in `WALLETS_CONFIG` let's use it for here as well.

```
case window.web3.currentProvider.isTrust:
          return WALLETS_CONFIG[WalletId.TRUST]
```